content/hi/docs/concepts/architecture/garbage-collection.md

```diff
@@ -0,0 +1,183 @@
+  * [बासी या समाप्त हो चुके सर्टिफिकेट साइनिंग अनुरोध (सीएसआर)](/docs/reference/access-authn-authz/certificate-signing-requests/#request-signing-process)
+  * {{<glossary_tooltip text="Nodes" term_id="node">}} निम्नलिखित परिदृश्यों में हटाया गया:
+    * क्लाउड पर जब क्लस्टर [क्लाउड कंट्रोलर मैनेजर](/docs/concepts/architecture/cloud-controller/)
+ का इस्तेमाल करता है
+    * ऑन-प्रिमाइसेस जब क्लस्टर क्लाउड कंट्रोलर के समान ऐडऑन का उपयोग करता है
+      प्रबंधक
+  * [नोड लीज ऑब्जेक्ट्स](/docs/concepts/architecture/nodes/#heartbeats)
+
+## स्वामी और आश्रित {#स्वामी-आश्रित}
+
+कुबेरनेट्स में कई वस्तुएं [*स्वामी संदर्भ*](/docs/concepts/overview/working-with-objects/owners-dependents/) के माध्यम से एक दूसरे से जुड़ती हैं।
+स्वामी के संदर्भ नियंत्रण तल को बताते हैं कि कौन सी वस्तुएं दूसरों पर निर्भर हैं।
+कुबेरनेट्स नियंत्रण विमान और अन्य एपीआई देने के लिए मालिक संदर्भों का उपयोग करता है
+ग्राहकों, हटाने से पहले संबंधित संसाधनों को साफ करने का अवसर
+वस्तु। ज्यादातर मामलों में, कुबेरनेट्स स्वचालित रूप से मालिक के संदर्भों का प्रबंधन करता है।
+
+स्वामित्व [लेबल और चयनकर्ता](/docs/concepts/overview/working-with-objects/labels/) से अलग है
+तंत्र जिसका कुछ संसाधन भी उपयोग करते हैं। उदाहरण के लिए, ए पर विचार करें
+{{<glossary_tooltip text="Service" term_id="service">}} जो बनाता है
+'एंडपॉइंटस्लाइस' ऑब्जेक्ट्स। सेवा नियंत्रण विमान को अनुमति देने के लिए *लेबल* का उपयोग करती है
+निर्धारित करें कि उस सेवा के लिए कौन से 'एंडपॉइंटस्लाइस' ऑब्जेक्ट का उपयोग किया जाता है। इसके साथ ही
+लेबल के लिए, प्रत्येक `EndpointSlice` जिसे किसी सेवा की ओर से प्रबंधित किया जाता है
+एक मालिक संदर्भ। मालिक के संदर्भ कुबेरनेट्स के विभिन्न भागों से बचने में मदद करते हैं
+उन वस्तुओं के साथ हस्तक्षेप करना जिन्हें वे नियंत्रित नहीं करते हैं।
+
+{{< नोट >}}
+क्रॉस-नेमस्पेस स्वामी संदर्भ डिज़ाइन द्वारा अस्वीकृत हैं।
+नेमस्पेस्ड आश्रित क्लस्टर-स्कोप्ड या नेमस्पेस्ड मालिकों को निर्दिष्ट कर सकते हैं।
+नामस्थान स्वामी **आवश्यक** उसी नामस्थान में आश्रित के रूप में मौजूद होना चाहिए।
+यदि ऐसा नहीं होता है, तो स्वामी संदर्भ को अनुपस्थित और आश्रित के रूप में माना जाता है
+एक बार सभी स्वामियों के अनुपस्थित होने की पुष्टि हो जाने के बाद उसे हटाया जा सकता है।
+
+क्लस्टर-दायरे वाले आश्रित केवल क्लस्टर-दायरे वाले स्वामियों को निर्दिष्ट कर सकते हैं।
+v1.20+ में, यदि एक क्लस्टर-दायरे वाला निर्भर एक स्वामी के रूप में एक नामस्थान प्रकार निर्दिष्ट करता है,
+इसे एक अनसुलझे मालिक संदर्भ के रूप में माना जाता है, और यह कचरा एकत्र करने में सक्षम नहीं है।
+
+v1.20+ में, अगर गारबेज संग्राहक एक अमान्य क्रॉस-नेमस्पेस `ओनर रेफरेंस` का पता लगाता है,
+या एक क्लस्टर-स्कोप्ड निर्भर एक `ओनर रेफरेंस` के साथ एक नेमस्पेस्ड प्रकार, एक चेतावनी ईवेंट को संदर्भित करता है
+'OwnerRefInvalidNamespace' के कारण और अमान्य आश्रित के 'शामिल ऑब्जेक्ट' की सूचना दी गई है।
+आप उस तरह के इवेंट को चलाकर चेक कर सकते हैं
+`kubectl get events -A --field-selector=reason=OwnerRefInvalidNamespace`।
+{{< /नोट >}}
```

Comment:
Avoid localizing the shortcode name to Hindi, as it results in [errors](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/65723d14589bf8000808beb6). Keep the shortcode name as it is and localize the content inside.

```suggestion
{{< note >}}
क्रॉस-नेमस्पेस स्वामी संदर्भ डिज़ाइन द्वारा अस्वीकृत हैं।
नेमस्पेस्ड आश्रित क्लस्टर-स्कोप्ड या नेमस्पेस्ड मालिकों को निर्दिष्ट कर सकते हैं।
नामस्थान स्वामी **आवश्यक** उसी नामस्थान में आश्रित के रूप में मौजूद होना चाहिए।
यदि ऐसा नहीं होता है, तो स्वामी संदर्भ को अनुपस्थित और आश्रित के रूप में माना जाता है
एक बार सभी स्वामियों के अनुपस्थित होने की पुष्टि हो जाने के बाद उसे हटाया जा सकता है।

क्लस्टर-दायरे वाले आश्रित केवल क्लस्टर-दायरे वाले स्वामियों को निर्दिष्ट कर सकते हैं।
v1.20+ में, यदि एक क्लस्टर-दायरे वाला निर्भर एक स्वामी के रूप में एक नामस्थान प्रकार निर्दिष्ट करता है,
इसे एक अनसुलझे मालिक संदर्भ के रूप में माना जाता है, और यह कचरा एकत्र करने में सक्षम नहीं है।

v1.20+ में, अगर गारबेज संग्राहक एक अमान्य क्रॉस-नेमस्पेस `ओनर रेफरेंस` का पता लगाता है,
या एक क्लस्टर-स्कोप्ड निर्भर एक `ओनर रेफरेंस` के साथ एक नेमस्पेस्ड प्रकार, एक चेतावनी ईवेंट को संदर्भित करता है
'OwnerRefInvalidNamespace' के कारण और अमान्य आश्रित के 'शामिल ऑब्जेक्ट' की सूचना दी गई है।
आप उस तरह के इवेंट को चलाकर चेक कर सकते हैं
`kubectl get events -A --field-selector=reason=OwnerRefInvalidNamespace`।
{{< /note >}}
```